### ISSUE TITLE
[appearance]Spaces not showing as brown dot with multiline special input

### ISSUE THREAD

**Issue by cymplecy** [jmoenig/Snap#2891]
![image](https://user-images.githubusercontent.com/2357428/133202420-d8ed63bb-0e9f-41b5-bd0d-214807d52bea.png)

Not a real problem with this example but bigger issue visually if you have leading or trailing spaces

**Comment by jmoenig** [comment]
yep, requiring whitespace visualization one of the main reasons why I'm not going for all-out multiline inputs
